src/js/utils/background.js

```diff
@@ -137,13 +182,32 @@ export const backgroundStyle = (backgroundArg, theme, textColorArg) => {
     theme,
   );
 
-  if (background.image) {
+  let backgroundImage = normalizeBackgroundImage(background, theme);
+
+  if (background.rotate) {
+    backgroundImage = rotateBackground(background, theme);
+  }
+
+  let backgroundClipStyle = '';
+  if (background.clip) {
+    backgroundClipStyle =
+      background.clip === 'text'
+        ? `-webkit-text-fill-color: transparent; 
+           -webkit-background-clip: text; 
+           background-clip: text;`
+        : `background-clip: ${background.clip};`;
+  }
+
+  if (backgroundImage) {
     const backgroundStyles = `
       ${backgroundColor ? `background-color: ${backgroundColor};` : ''}
-      background-image: ${background.image};
-      background-repeat: ${background.repeat || 'no-repeat'};
+      background-image: ${backgroundImage};
+      background-repeat: ${
+        (typeof background === 'object' && background.repeat) || 'no-repeat'
```

Comment:
```suggestion
        background?.repeat || 'no-repeat'
```

Reply:
I investigated this. The original reason for the `typeof === 'object'` is because if `background` is a string, it is getting interpreted as the repeat method on `String`.